src/main/webapp/app/exercises/programming/participate/code-editor-student-container.component.html

```diff
@@ -35,7 +35,7 @@
                 ></jhi-included-in-score-badge
             ></span>
         </div>
-        <div submissionPolicy *ngIf="exercise.submissionPolicy && numberOfSubmissionsForSubmissionPolicy !== undefined">
+        <div submissionPolicy *ngIf="exercise.submissionPolicy && exercise.submissionPolicy.active && numberOfSubmissionsForSubmissionPolicy !== undefined">
```

Comment:
Probably, numberOfSubmissionsForSubmissionPolicy !== undefined can be also replaced as !!numberOfSubmissionsForSubmissionPolicy

Reply:
No, since 0 is also considered falsy, but should be displayed here